src/actions/voting.test.js

```diff
@@ -213,7 +214,7 @@ describe('actions: voting', () => {
       peers: {
         data: {
           options: {
-            name: 'Mainnet',
+            name: Networks.mainnet.name,
```

Comment:
I would use only capital letters to name a variable if and only if its a react component or a class.